addons/hr_timesheet/tests/test_project_update.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+from odoo.addons.project.tests.test_project_update import TestProjectUpdate
+from odoo.tests import tagged
+from odoo.tests.common import Form
```

Comment:
Based on [guidelines](https://www.odoo.com/documentation/master/reference/guidelines.html#imports), we should do this: 
```suggestion
from odoo.tests import tagged
from odoo.tests.common import Form

from odoo.addons.project.tests.test_project_update import TestProjectUpdate
```